tardis/apps/publication_workflow/static/js/lib/angular-material-data-table-0.10.10/dist/md-data-table.js

```diff
@@ -0,0 +1,1486 @@
+        angular.element(cell).empty();
+      }
+    }
+    
+    function enableRowSelection() {
+      return tableCtrl.$$rowSelect;
+    }
+    
+    function mdSelectCtrl(row) {
+      return angular.element(row).controller('mdSelect');
+    }
+    
+    function removeCheckboxColumn() {
+      Array.prototype.some.call(element.find('th'), function (cell) {
+        return cell.classList.contains('md-checkbox-column') && cell.remove();
+      });
+    }
+    
+    scope.allSelected = function () {
+      var rows = scope.getSelectableRows();
+      
+      return rows.length && rows.every(function (row) {
+        return row.isSelected();
+      });
+    };
+    
+    scope.getSelectableRows = function () {
+      return tableCtrl.getBodyRows().map(mdSelectCtrl).filter(function (ctrl) {
+        return ctrl && !ctrl.disabled;
+      });
+    };
+    
+    scope.selectAll = function () {
+      tableCtrl.getBodyRows().map(mdSelectCtrl).forEach(function (ctrl) {
+        if(ctrl && !ctrl.isSelected()) {
+          ctrl.select();
+        }
+      });
+    };
+    
+    scope.toggleAll = function () {
+      return scope.allSelected() ? scope.unSelectAll() : scope.selectAll();
+    };
+    
+    scope.unSelectAll = function () {
+      tableCtrl.getBodyRows().map(mdSelectCtrl).forEach(function (ctrl) {
+        if(ctrl && ctrl.isSelected()) {
+          ctrl.deselect();
+        }
+      });
+    };
+    
+    scope.$watchGroup([enableRowSelection, tableCtrl.enableMultiSelect], function (newValue) {
+      if(newValue[0] !== oldValue[0]) {
+        if(newValue[0]) {
+          addCheckboxColumn();
+          
+          if(newValue[1]) {
+            attatchCheckbox();
+          }
+        } else {
+          removeCheckboxColumn();
+        }
+      } else if(newValue[0] && newValue[1] !== oldValue[1]) {
+        if(newValue[1]) {
+          attatchCheckbox();
+        } else {
+          detachCheckbox();
+        }
+      }
+      
+      angular.copy(newValue, oldValue);
+    });
+  }
+  
+  return {
+    bindToController: true,
+    compile: compile,
+    controller: Controller,
+    controllerAs: '$mdHead',
+    require: '^^mdTable',
+    restrict: 'A',
+    scope: {
+      order: '=?mdOrder',
+      onReorder: '=?mdOnReorder'
+    }
+  };
+}
+
+mdHead.$inject = ['$compile'];
+
+angular.module('md.data.table').directive('mdRow', mdRow);
+
+function mdRow() {
+
+  function compile(tElement) {
+    tElement.addClass('md-row');
+    return postLink;
+  }
+  
+  function postLink(scope, element, attrs, tableCtrl) {
+    function enableRowSelection() {
+      return tableCtrl.$$rowSelect;
+    }
+    
+    function isBodyRow() {
+      return tableCtrl.getBodyRows().indexOf(element[0]) !== -1;
+    }
+    
+    function isChild(node) {
+      return element[0].contains(node[0]);
+    }
+    
+    if(isBodyRow()) {
+      var cell = angular.element('<td class="md-cell">');
+      
+      scope.$watch(enableRowSelection, function (enable) {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 16 spaces but found 6.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
                scope.$watch(enableRowSelection, function (enable) {
```